labscript_devices/AD9959DDSSweeper/blacs_tabs.py

```diff
@@ -0,0 +1,56 @@
+#####################################################################
+#                                                                   #
+# /labscript_devices/AD9959DDSSweeper/blacs_tabs.py                 #
+#                                                                   #
+# Copyright 2025, Carter Turnbaugh                                  #
+#                                                                   #
+# This file is part of the module labscript_devices, in the         #
+# labscript suite (see http://labscriptsuite.org), and is           #
+# licensed under the Simplified BSD License. See the license.txt    #
+# file in the root of the project for the full license.             #
+#                                                                   #
+#####################################################################
+
+from blacs.device_base_class import DeviceTab
+
+class AD9959DDSSweeperTab(DeviceTab):
+    def initialise_GUI(self):
+        # Capabilities
+        self.base_units =    {'freq':'Hz',          'amp':'Arb',   'phase':'Degrees'}
+        self.base_min =      {'freq':0.0,           'amp':0,       'phase':0}
+        self.base_max =      {'freq':250.0*10.0**6, 'amp':1,       'phase':360}
```

Comment:
We should probably set `self.base_max` based on the connection table properties (`device.properties['ref_clock_frequency']` and `device.properties['pll_mult']`).